etc/spack/defaults/config.yaml

```diff
@@ -66,3 +66,14 @@ config:
   # If set to true, `spack install` and friends will NOT clean
   # potentially harmful variables from the build environment. Use wisely.
   dirty: false
+
+
+  # If set to true, `spack install` and friends will run `make` in parallel.
+  # If set to false, all builds will run in serial.
+  parallel: true
```

Comment:
Can we simplify this and just have one setting for `jobs`?  If `jobs` is 1, everything's serial.  If not, it's parallel.  I realize this corresponds to the arguments that `make` currently takes, but there is a subtle difference.  `make` takes `parallel=True` or `parallel=False` because the only thing the packager should care about is whether or not the build *can* be parallel.  It's up to the user/admin of a particular Spack instance to decide how many jobs there are.